src/ast/visitor.rs

```diff
@@ -884,4 +884,29 @@ mod tests {
             assert_eq!(actual, expected)
         }
     }
+
+    struct QuickVisitor; // [`TestVisitor`] is too slow to iterate over thousands of nodes
+
+    impl Visitor for QuickVisitor {
+        type Break = ();
+    }
+
+    #[test]
+    fn overflow() {
+        let cond = (0..1000)
```

Comment:
Same reason as `sqlparser_common::overflow` test. Maybe we could increase it(increase the number `1000`).